city_scrapers/spiders/chi_ssa_27.py

```diff
@@ -0,0 +1,123 @@
+from datetime import datetime
+
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa27Spider(CityScrapersSpider):
+    name = "chi_ssa_27"
+    agency = "Chicago Special Service Area #27 Lakeview West"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.lakeviewchamber.com"]
+    start_urls = ["https://www.lakeviewchamber.com/ssa27"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        meeting = None
+        meet_path = "/html/body/div/div[2]/main/div/div[2]/div[2]" \
+                    "/div[2]/ul/li[1]/div/div/div[2]/p/text()"
+        response_xpath = response.xpath(meet_path).getall()
+        for item in response_xpath:
+            if "Annual" not in item:
+                meeting = Meeting(
+                    title=self._parse_title(),
+                    description=self._parse_description(),
+                    classification=self._parse_classification(),
+                    start=self._parse_start(item),
+                    end=self._parse_end(),
+                    all_day=self._parse_all_day(),
+                    time_notes=self._parse_time_notes(),
+                    location=self._parse_location(),
+                    links=self._parse_links(),
+                    source=self._parse_source(response),
+                )
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+        yield meeting
+
+    @staticmethod
+    def _parse_title():
+        """Parse or generate meeting title."""
+        meeting_title = "SSA 27 Commission Meeting"
```

Comment:
We can just use "Commission" here, but if it's a committee meeting it should return the name of the committee instead.